### ISSUE TITLE
Add React Router

### ISSUE THREAD

**Issue by amelv** [amelv/birthday-atlas#4]
We need React Router since we need routes in this application. But why?

While not requested by the project assignment specs, I felt having a landing page separate from the actual birthday profile explore dashboard was a good idea. So we have at least 2 routes. Maybe more if I have time to add more.

Since I intend to make this app as close to how I would generally design an application, I feel a landing page introduces users to the purpose of the application. It calls them to use with that purpose in mind. We want them to feel invited to a tool, not thrown into the works immediately. 

So, to put it simply, it's polite UI design to have a landing page 😆
